src/net/conn.c

```diff
@@ -46,7 +47,7 @@ ts_connection_create(ConnectionType type)
 
 	conn = connection_internal_create(type, conn_ops[type]);
 
-	Assert(NULL != conn);
+	Ensure(conn, "unable to create connection");
```

Comment:
Hmm this seems like a bug here cause connection_internal_create can return NULL and should therefore always be checked.

Reply:
Hmm.... I'll take a separate PR to check all usage of `connection_internal_create` but leave this for now (it will error out in release builds, which is the important part).